site/en/tutorials/quickstart/advanced.ipynb

```diff
@@ -165,7 +165,7 @@
         "id": "BPZ68wASog_I"
       },
       "source": [
-        "Build the `tf.keras` model using the Keras [model subclassing API](https://www.tensorflow.org/guide/keras#model_subclassing):"
+        "Build the `tf.keras` model using the Keras [model subclassing API](https://www.tensorflow.org/guide/keras/custom_layers_and_models#the_model_class):"
```

Comment:
Thanks @RenuPatelGoogle 

Since the section title may change in future and then break the link, let's just use the main URL here.

Reply:
```suggestion
        "Build the `tf.keras` model using the Keras [model subclassing API](https://www.tensorflow.org/guide/keras/custom_layers_and_models):"
```